packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts

```diff
@@ -0,0 +1,596 @@
+  expect(auctionsAfter[0].batches[0].loan).toStrictEqual('500.00399539@TSLA')
+}
+
+describe('placeAuctionBid success', () => {
+  beforeEach(async () => {
+    await tGroup.start()
+    await alice.container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterEach(async () => {
+    await tGroup.stop()
+  })
+
+  it('should placeAuctionBid', async () => {
+    // test end the auction without bid
+    await bob.generate(36)
+    {
+      const auctions = await bob.container.call('listauctions')
+      expect(auctions[0].liquidationHeight).toStrictEqual(205) // liquidationHeight updated
+    }
+
+    // cont. test auctionBid
+    const bobColAccBefore = await bob.rpc.account.getAccount(bobColAddr)
+    expect(bobColAccBefore).toStrictEqual(['8900.00000000@DFI', '545.45454546@TSLA'])
+
+    const bobTSLAAccBefore = bobColAccBefore.length > 0
+      ? bobColAccBefore.find((amt: string) => amt.split('@')[1] === 'TSLA')
+      : undefined
+    const bobTSLAAmtBefore = bobTSLAAccBefore !== undefined ? Number(bobTSLAAccBefore.split('@')[0]) : 0
+
+    const txid = await bob.rpc.loan.placeAuctionBid({
+      vaultId: bobVaultId,
+      index: 0,
+      from: bobColAddr,
+      amount: '526@TSLA' // (500.00399539 * 5%) + 500 = 525.00419516, min first bid is 525.00419516
+    })
+    expect(typeof txid).toStrictEqual('string')
+    await bob.container.generate(1)
+    await tGroup.waitForSync()
+
+    const bobColAccAfter = await bob.rpc.account.getAccount(bobColAddr)
+    expect(bobColAccAfter).toStrictEqual(['8900.00000000@DFI', '19.45454546@TSLA'])
+
+    const bobTSLAAccAfter = bobColAccAfter.length > 0
+      ? bobColAccAfter.find((amt: string) => amt.split('@')[1] === 'TSLA')
+      : undefined
+    const bobTSLAAmtAfter = bobTSLAAccAfter !== undefined ? Number(bobTSLAAccAfter.split('@')[0]) : 0
+
+    expect(bobTSLAAmtBefore - bobTSLAAmtAfter).toStrictEqual(526) // 545.45454546 - 19.45454546 = 526
+
+    const aliceColAccBefore = await alice.rpc.account.getAccount(aliceColAddr)
+    expect(aliceColAccBefore).toStrictEqual(['30000.00000000@DFI', '29999.00000000@BTC', '10000.00000000@TSLA'])
+
+    // test second round placeAuctionBid
+    await alice.rpc.loan.placeAuctionBid({
+      vaultId: bobVaultId,
+      index: 0,
+      from: aliceColAddr,
+      amount: '535@TSLA' // (526 * 1%) + 525 = 531.26
+    })
+    await alice.generate(1)
+    await tGroup.waitForSync()
+
+    // bob should be able to claim back his fund right after the higher bid
+    const bobColAccBid = await bob.rpc.account.getAccount(bobColAddr)
+    expect(bobColAccBid).toStrictEqual(['8900.00000000@DFI', '545.45454546@TSLA'])
+
+    // let the auction end and the vault's state will be switched to mayLiquidate
+    await bob.container.waitForVaultState(bobVaultId, VaultState.MAY_LIQUIDATE)
+    // after convert from mayLiquidate, gen(1) will cause interest increase
+    await bob.container.waitForVaultState(bobVaultId, VaultState.IN_LIQUIDATION)
```

Comment:
@surangap 
here's the cause about increasing interest by 1 block in auction bid loan